### ISSUE TITLE
Display footnotes at bottom of page instead of inline

### ISSUE THREAD

**Issue by mojavelinux** [asciidoctor/asciidoctor-pdf#73]
Display footnotes at the bottom of the page instead of inline.

**Comment by mojavelinux** [comment]
I think for 1.5.0 we can start by displaying them at the end of the chapter. Putting them at the bottom of the page is very difficult task.

**Comment by rlopez133** [comment]
Excited to see this in the beta milestones with the next tag! :+1:

**Comment by Gitward** [comment]
Displaying footnotes seems to work when creating the PDF using https://github.com/asciidocfx/AsciidocFX . What have they done to solve the problem?

My Test-document:

```
= Test
:doctype: article
:notitle:
:!toc:

AsciidocFX shows links in PDFs as footnotes http://stackoverflow.com[SO].

.Asciidoc in PDF does not work in Asciidoctor, but works in AsciidocFX.
[cols="2,5a"]
|===
|Line with Asciidoc code
|here comes a list:

* item 1
* item 2
* item 3

http://stackoverflow.com[Get Answers]!

|Line
|with a footnotefootnote:[footnotes do work in AsciidocFX's PDF output (but not in the preview).]

|===
```
The output of asciidoctor (using gradle):

![asciidoctor](https://cloud.githubusercontent.com/assets/8510849/15852939/532ac8de-2ca3-11e6-8908-821c0bba0895.png)

The output of asciidocFX:

![asciidocfx](https://cloud.githubusercontent.com/assets/8510849/15852949/5ac9f6be-2ca3-11e6-943e-47ba275c07bd.png)

**Comment by mojavelinux** [comment]
AsciiDocFX is not yet using Asciidoctor PDF. Instead, it uses the DocBook toolchain. It's essentially using [fopub](https://github.com/asciidoctor/asciidoctor-fopub) under the covers, which executes and manages the DocBook toolchain.

**Comment by PierreR** [comment]
I was so disappointed when I realized this was not yet implemented ... And would be so happy if it comes up soonish ;-)

**Comment by mojavelinux** [comment]
I'm essentially proposing that we start with treating footnotes as endnotes. I think that's a good first step.

**Comment by mojavelinux** [comment]
For the book doctype, the footnotes should be placed at the end of the chapter. For the article doctype, the footnotes should be placed at the end of the document.

As each footnote is visited, create and insert an anchor point next to the footnote number and store the entry in a table. When a new chapter is started (book doctype only), or the end of the document is reached, back up a page and place the footnotes at the bottom of the page (or on next page if not enough space is available on the previous page). It should be possible to jump back and forth between the footnote number and the footnote entry by clicking on the number.

**Comment by mojavelinux** [comment]
Note that this is a duplicate of #85.

**Comment by mojavelinux** [comment]
As soon as I get back to working on Asciidoctor PDF, I'll dive into this issue. I got called away in Jan for a big job and have been a bit behind.

**Comment by barspi** [comment]
Eagerly awaiting the release of alpha.14 :-)

**Comment by mescobal** [comment]
Any news about this bug? The ONE thing that I need to continue writing a book using asciidoc!!!!!!

**Comment by mojavelinux** [comment]
I would advise following this issue, which is where the main discussion is happening.https://github.com/asciidoctor/asciidoctor-pdf/issues/85#issuecomment-374121410

I'm going to mark this issue as a duplicate so we can consolidate the conversation.

**Comment by mojavelinux** [comment]
We decided to make them end notes. Solved by the fix for #85.
